src/config.py

```
@@ -0,0 +1,90 @@
+
+class RetryableCredentials(_BaseCreds):
+    @tenacity.retry(
+        stop=tenacity.stop_after_attempt(5),
+        wait=tenacity.wait_exponential(min=1, max=10),
+        reraise=True
+    )
+    def refresh(self, request: AuthRequest) -> None:
+        return super().refresh(request)
+
+# Configuration constants
+DB_PATH = Path(os.environ.get("DB_FILEPATH", "~/Library/Messages/chat.db")).expanduser()
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 1

(E305)